pkgs/top-level/all-packages.nix

```diff
@@ -27989,6 +27989,16 @@ with pkgs;
 
   prusa-slicer = callPackage ../applications/misc/prusa-slicer { };
 
+  snapmaker-luban = callPackage ../applications/misc/snapmaker-luban {
```

Comment:
This section seems to cover prusa slicer tools so I'd probably move this somewhere else

Reply:
It's moved now.